qualtrics-event-gw/kubernetes/registrationfile.json

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
Please move that to the related connector bundle

Reply:
removed